tools/packaging/kernel/configs/fragments/x86_64/crypto.conf

```diff
@@ -0,0 +1,3 @@
+# x86 cryptographic instructions to improve AES encryption and SHA256 hashing.
```

Comment:
@fidencio @arronwy How do we handle config options for non-x86 architectures? Add them here or separate commit in this PR or additional PR?

cc: @stevenhorsman @BbolroC

Reply:
@arronwy thanks. Looking on the `s390x` the are a couple of config options already in-place. I will check for those which has been added in the last 2y. So let me know whether to follow-up here or separately?

Reply:
Yes, like `arm` also already have some crypto accelerated config, I think we can also open an PR for `s390x` if needed